### ISSUE TITLE
Add Scully CLI to build image

### ISSUE THREAD

**Issue by hisham** [aws-amplify/amplify-console#1148]
**Is your feature request related to a problem? Please describe.**
I see the default build image that comes with Amplify comes with Gatsby and Hugo. 

Scully is an up and coming static site generator for Angular. We plan to use it soon. It would be nice if Amplify Console team considers adding it to the default build image. It is new and v1 was just released so perhaps not as popular yet as Gatsby and Hugo. But just putting this request here for consideration since it's the only static site generator option for Angular at this time.

In the meantime, my team will experiment with having a custom build image. 

**Additional context**
- Scully website: https://scully.io/
- Example dockerfile: https://hub.docker.com/r/beeman/scully-docker/dockerfile

**Comment by swaminator** [comment]
@hirochachacha thanks for your request. Happy to hear from the community to see if the demand is there.

**Comment by mfeitoza** [comment]
@hisham any success to setup Scully in Amplify?

**Comment by hisham** [comment]
yeah we were able to set it up. Had to use our own custom docker image ofcourse.

**Comment by mfeitoza** [comment]
Could you share the Dockerfile?

**Comment by mkreder** [comment]
I have created an image for this and uploaded it to docker hub. It is  "mkreder/amplify-scully"

DockerHub
https://hub.docker.com/r/mkreder/amplify-scully

Github
https://github.com/mkreder/amplify-scully
